[2024-04-15T09:10:43.379+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_data.extract_data_from_text manual__2024-04-15T09:10:41.211655+00:00 [queued]>
[2024-04-15T09:10:43.387+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_data.extract_data_from_text manual__2024-04-15T09:10:41.211655+00:00 [queued]>
[2024-04-15T09:10:43.388+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-04-15T09:10:43.400+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): extract_data_from_text> on 2024-04-15 09:10:41.211655+00:00
[2024-04-15T09:10:43.405+0000] {standard_task_runner.py:60} INFO - Started process 532 to run task
[2024-04-15T09:10:43.408+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_data', 'extract_data_from_text', 'manual__2024-04-15T09:10:41.211655+00:00', '--job-id', '1449', '--raw', '--subdir', 'DAGS_FOLDER/etl_data.py', '--cfg-path', '/tmp/tmphta_3_p9']
[2024-04-15T09:10:43.410+0000] {standard_task_runner.py:88} INFO - Job 1449: Subtask extract_data_from_text
[2024-04-15T09:10:43.447+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_data.extract_data_from_text manual__2024-04-15T09:10:41.211655+00:00 [running]> on host 41da1b14e620
[2024-04-15T09:10:43.508+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_data' AIRFLOW_CTX_TASK_ID='extract_data_from_text' AIRFLOW_CTX_EXECUTION_DATE='2024-04-15T09:10:41.211655+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-15T09:10:41.211655+00:00'
[2024-04-15T09:10:43.510+0000] {python.py:202} INFO - Done. Returned value was: file does not exist
[2024-04-15T09:10:43.655+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=etl_data, task_id=extract_data_from_text, execution_date=20240415T091041, start_date=20240415T091043, end_date=20240415T091043
[2024-04-15T09:10:43.701+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-15T09:10:43.725+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
